documentation/docs/liquidity-mining/faq.md

```diff
@@ -28,20 +33,25 @@ In addition, the prohibitively high payment demanded by pro market makers, coupl
 
-## What risks does a market maker bear?
+## What risks does a liquidity miner bear?
 Like any trading strategy, market making includes risk. One of the primary risks is **inventory risk**, the risk of negative changes in inventory value as a result of market making. For instance, if prices drop significantly in a short time period and a market maker accumulates a large position in the asset due to continual fills of their market maker's buy orders, their overall inventory value may be lower.
 
-## How do you verify that liquidity bounty trading volume is real?
-We take compliance extremely seriously, because we want to reward only genuine providers of liquidity. For this reason, participants need to opt into data collection, which allows us to attempt to verify their trading activity against our internal exchange data feeds in order to prevent reporting of fake volume.
+## How do you determine liquidity miners' rewards?
+In order to make economic sense for a market maker, the market maker’s compensation must correlate with increased levels of risk. There are three main parameters that we use in liquidity mining to determine market maker compensation: (1) **time**: placing orders in the order book consistently over time, (2) **spreads**, and (3) **order sizes**. Our rewards methodology rewards market makers more for placing orders consistently over time in the order book, placing orders with tighter spreads and with larger sizes as shown in the below figure. The real-time reward information will be displayed in the real-time user dashboard. 
+![](https://hummingbot.io/static/e9bcf5a0b0ad5320f95f0a1de89c3e9a/ed7b0/rewards-allocation-chart.png)
+
+For more details on the calculation, please read [Demystifying Liquidity Mining Rewards](https://hummingbot.io/blog/2019-12-liquidity-mining-rewards/). 
 
-In addition, we will run proprietary algorithms in order to attempt to detect wash trading and spoofing. While exploitative practices can be difficult to identify given the adversarial nature of the market, we believe that the combination of our focus on compliance, granular data feeds, and machine learning-based algorithms may deter and detect bad actors.
+## How do you verify the trading activities?
+We take compliance extremely seriously, because we want to reward only genuine providers of liquidity. For this reason, participants need to opt into data collection and provide their read-only API keys on exchanges, allowing us to attempt to verify their trading activity. In addition, we will run proprietary algorithms in order to attempt to detect wash trading and spoofing. While exploitative practices can be difficult to identify given the adversarial nature of the market, we believe that the combination of our focus on compliance, granular data feeds, and machine learning-based algorithms may deter and detect bad actors.
```

Comment:
We take compliance extremely seriously, and only reward genuine providers of liquidity. For this reason, participants need to opt into data collection and provide their read-only API keys for exchanges, in order to allow us to verify trading activity. In addition, we run proprietary algorithms in order to attempt any prohibited actions such as wash trading and spoofing. While exploitative practices can be difficult to identify given the adversarial nature of the market, we believe that the combination of our focus on compliance, granular data feeds, and machine learning-based algorithms may deter and detect bad actors.